test/tests/performance/pbs_client_nagle_performance.py

```diff
@@ -50,21 +51,29 @@ class TestClientNagles(TestPerformance):
 
     def setUp(self):
         """
-            Base class method overridding
-            builds absolute path of commands to execute
+        Base class method overriding
+        builds absolute path of commands to execute
         """
         TestPerformance.setUp(self)
         self.time_command = self.du.which(exe="time")
         if self.time_command == "time":
             self.skipTest("Time command not found")
 
+    def tearDown(self):
+        """
+        cleanup jobs
+        """
+
+        TestPerformance.tearDown(self)
+        self.server.cleanup_jobs(runas=ROOT_USER)
```

Comment:
I assumed that too but I did not see it happening. They were getting cleaned at setup but sometime that was causing timeout hence I moved this to tearDown.